CHANGES_NEXT_RELEASE

```diff
@@ -7,3 +7,24 @@
 - Update COAP librery to version 0.21.0
 - Using precise dependencies (~=) in packages.json
 - Update MongoDB to 2.2.35
+- Fix: updating dependencies due to known vulnerabilities in the previous ones
+	async: 1.5.2 -> 2.6.1
+    underscore: 1.7.0 -> 1.9.1
+    logops: 0.5.0 -> 1.0.0
+    coap: 0.5.0 -> 1.0.0
+    coap-cli: 0.5.0 -> 1.0.0
+    mongodb: 2.2.35 -> 3.1.8
+    mongoose: 4.4.3 -> 5.3.9
+- Remove: unmaintained dependency with mongoose-auto-increment
```

Comment:
I understand mongo-plugin-autoinc is a kind of replacement of mongoose-auto-increment. As alternative you could replace these 2 lines by:

```
- Replace mongoose-auto-increment dependency (unmantained) with mongoose-plugin-autoinc
```

Reply:
Fixed with 18fc9c6da0ff51552ac0068654c017b953354440